src/rust/sumologic-mock/src/main.rs

```diff
@@ -150,7 +150,7 @@ async fn run_app(hostname: String, port: u16, opts: Options) -> std::io::Result<
         fields: Mutex::new(HashMap::new()),
     });
 
-    info!("Receiver mock is waiting for enemy on 0.0.0.0:{}!", port);
+    info!("Sumo Logic Mock is listening on 0.0.0.0:{}!", port);
```

Comment:
I've liked that part :(

Reply:
I used to be `Sumock is waiing for enemy` before we agreed to use `receiver-mock` name

I just made it public and archive: https://github.com/sumo-drosiek/sumock